CHANGELOG_PENDING.md

```diff
@@ -22,4 +22,5 @@ Special thanks to external contributors on this release:
 
 ### BUG FIXES:
 - [kv indexer] \#2912 don't ignore key when executing CONTAINS
+- [mempool] \#2994 Don't allow txs with negative gas wanted
```

Comment:
All this PR does now is adding the comment. Should we remove the changelog entry then?

```suggestion
```

Reply:
That was from a previous pr, forgot to add that entry